src/construction.cpp

```diff
@@ -136,6 +136,36 @@ void draw_grid( WINDOW *w, const int list_width = 30 )
     wrefresh( w );
 }
 
+nc_color construction_color( std::string &con_name, bool highlight )
+{
+    nc_color col = c_dkgray;
+    if( g->u.has_trait( "DEBUG_HS" ) ) {
+        col = c_white;
+    } else if( can_construct( con_name ) ) {
+        construction *con_first = NULL;
```

Comment:
Not a criticism (I see, the snippet of code was moved), but it would be nice to update the code a bit during moving it to the new place. `NULL` macro is essentially a number (zero). Please, replace it with `nullptr` which is of pointer type and preferable since c++11.